Enabling the use of SNMPv3 traps (#216)
librenms/docker: doc/docker/environment-variables.md

> :warning: Only used if you enable and run a [sidecar snmptrapd container](../notes/snmptrapd.md)

* `SIDECAR_SNMPTRAPD`: Set to `1` to enable sidecar snmptrapd mode for this container (default `0`)

> * `SNMP_ACTION`: Sets what actions (`log`, `execute`, and/or `net`) to take with the SNMP trap (default `log,execute,net`)

Author: ```suggestion
* `SNMP_PROCESS_TYPE`: Sets which type of processing (`log`, `execute`, and/or `net`) to take with the SNMP trap (default `log,execute,net`)
```
